README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
```

Comment:
delete this or ill delete your firstborn child

https://www.google.com/url?sa=i&source=images&cd=&ved=2ahUKEwjMzYna1d_kAhUIWN8KHRoGCGAQjRx6BAgBEAQ&url=https%3A%2F%2Fwww.dreamstime.com%2Fstock-photo-old-woman-angry-gesture-making-fists-white-background-image41310996&psig=AOvVaw2udH3m2q32Tp9LNj-QKOyd&ust=1569078121161531